tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -25,8 +25,9 @@
     "dcv_port, access_from, shared_dir", [(8443, "0.0.0.0/0", "/shared"), (5678, "192.168.1.1/32", "/myshared")]
 )
 @pytest.mark.regions(["eu-west-1", "cn-northwest-1"])  # DCV license bucket not present in us-gov
-@pytest.mark.oss(["centos7"])
-@pytest.mark.schedulers(["sge", "awsbatch"])
+@pytest.mark.oss(get_supported_dcv_os())
```

Comment:
You need to rebase the code, let's use "centos7" and not the "get_supported_dcv_os" function to split code and tests

Reply:
Done.